lang/swi-prolog-devel/Portfile

```diff
@@ -32,8 +32,8 @@ distname            swipl-${version}
 dist_subdir         swi-prolog
 
 checksums           rmd160  6d79ef869091fe6f6664bf3f1fd064f2fe8a97b8 \
-                    sha256  12d51d38633e733ca42d69bc02b3b844e40592212ac41511797ab6afa1a9b812 \
-                    size    10955310
+                    sha256  444558321c34b9c75fe1ab81057038b6799285dec5055ab5c285f78ec43929f3 \
+                    size    10956733
```

Comment:
the `rmd160` cannot be the same if `sha256` and `size` change and that's why the build is currently failing.